src/App.css

```diff
@@ -1,87 +1,264 @@
+  gap: 0.8em;
 }
 
 .bookDetails {
   display: flex;
   flex-direction: column;
 }
 
-.author {
+.category {
+  color: var(--black-two);
+  opacity: 0.5;
+  font-family: var(--font-mont);
   font-weight: bold;
-  margin-bottom: 5px;
+  font-size: 0.875rem;
+  padding: 0;
+  margin: 0;
 }
 
-.removeBookButton {
-  background-color: #ff4c4c;
-  color: #fff;
+.title {
+  color: var(--black-two);
+  opacity: 0.9;
+  font-size: 1.375rem;
+  font-weight: bold;
+  font-family: var(--font-roboto);
+  line-height: normal;
+  padding: 5px 0;
+  margin: 0;
+}
+
+.author {
+  font-family: var(--font-roboto);
+  font-size: 0.875rem;
+  font-weight: 300;
+  color: var(--cool-blue);
+  padding: 0;
+  margin: 0;
+}
+
+.actionBtn {
+  color: var(--cool-blue);
   border: none;
+  background-color: var(--white);
+  padding: 0 1em;
+  font-family: var(--font-roboto);
+  font-size: 0.875rem;
+  font-weight: 300;
+}
+
+.comment {
+  padding-left: 0;
+}
+
+.removeBook {
+  border-left: 1px solid #0290ffa3;
+  border-right: 1px solid #0290ffa3;
+}
+
+.percentage {
+  display: flex;
+  align-items: center;
+  padding-left: 10%;
+  padding-right: 4.5em;
+  border-right: 1px solid #38373741;
+}
+
+.progressBar {
+  height: 60px;
+  width: 10vw;
+}
+
+.numberPercentage {
+  color: var(--black-two);
+  padding: 0.5em;
+}
+
+.numberPercentage span {
+  padding: 0;
+  margin: 0;
+  font-size: 25px;
+  font-family: var(--font-mont);
+}
+
+.numberPercentage p {
+  padding: 0;
+  margin: 0;
+  font-family: var(--font-mont);
+}
+
+.currentChapter {
+  font-size: 0.813rem;
+  font-weight: 300;
+  font-family: var(--font-roboto);
+  color: var(--black-two);
+  opacity: 0.5;
+}
+
+.chapter {
+  font-family: var(--font-roboto);
+  color: var(--black-two);
+  font-size: 1rem;
+  font-weight: 300;
+}
+
+.updateProgress {
+  padding: 0.438rem 1.188rem 0.5rem 1.375rem;
+  border-radius: 3px;
+  background-color: var(--azure);
+  border: none;
+  color: var(--white);
+  font-size: 0.813rem;
+  font-weight: 300;
+  font-family: var(--font-roboto);
+}
+
+.newBook {
+  padding: 10px;
+}
+
+.addNewBook {
+  font-family: var(--font-mont);
+  color: #888;
+  font-size: 1.25rem;
+  font-weight: bold;
+  padding: 5px 1.8em;
+  margin: 0;
+}
+
+form {
+  display: flex;
+  gap: 1.5em;
+  padding: 10px 0;
+  margin: 0 2em;
+  align-items: center;
+  width: 100%;
+}
+
+input[type="text"] {
+  height: 3em;
+  width: 35.3%;
+  min-width: 250px;
   border-radius: 4px;
-  padding: 5px 10px;
-  cursor: pointer;
-  font-size: 14px;
+  border: solid 1px #e8e8e8;
 }
 
-.noBooksMessage {
-  color: #aaa;
+input[type="submit"] {
+  font-size: 0.813rem;
+  font-weight: bold;
+  font-family: var(--font-roboto);
+  color: var(--pale-grey);
+  background-color: var(--azure);
+  border: none;
+  padding: 0.801rem 1.188rem 0.886rem 1.375rem;
+  border-radius: 3px;
+  margin: 0;
   text-align: center;
+  width: 11.5rem;
+  height: 2.5rem;
+}
+
+input[type="submit"]:hover {
+  background-color: #f8f8f8;
 }
```

Comment:
- [ ] It would be nicer if you could work on aligning the form input fields to be inline from top to bottom just as it's well aligned in the original design. It's required that your design match the original design and you're one step there. Kindly refer to the screenshots for better arrow demos 👍🏽

**The Original Design**

<img width="1183" alt="Screenshot 2023-04-08 at 6 32 19 PM" src="https://user-images.githubusercontent.com/50941074/230737502-5b492f3f-053e-4f20-8d41-184d34bc9eb9.png">

**Your Design**

<img width="1660" alt="Screenshot 2023-04-08 at 6 38 04 PM" src="https://user-images.githubusercontent.com/50941074/230737669-be2748d9-99f2-4bfb-8138-fe95dbb92f3a.png">

Reply:
Done

Reply:
Fixed.